iris-backend-service/src/main/java/iris/backend_service/alerts/zammad/ZammadTicketCreator.java

```diff
@@ -0,0 +1,81 @@
+package iris.backend_service.alerts.zammad;
+
+import iris.backend_service.alerts.Alert;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.MediaType;
+import org.springframework.http.client.ClientHttpRequestExecution;
+import org.springframework.http.client.ClientHttpRequestInterceptor;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * @author Jens Kutzsche
+ */
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ZammadTicketCreator {
+
+	private final RestTemplate rest;
+	private final ZammadProperties properties;
+
+	@PostConstruct
+	void postConstruct() {
+
+		rest.getInterceptors().add(new ClientHttpRequestInterceptor() {
+
+			@Override
+			public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttpRequestExecution execution)
+					throws IOException {
+
+				var headers = request.getHeaders();
+				headers.setContentType(MediaType.APPLICATION_JSON);
+				headers.setAccept(List.of(MediaType.APPLICATION_JSON));
+				headers.add(HttpHeaders.AUTHORIZATION, "Bearer " + properties.getToken());
+
+				return execution.execute(request, body);
+			}
+		});
+	}
+
+	public boolean createTicket(Alert alert) {
+
+		if ("DISABLED".equals(properties.getToken())) {
+			return true;
+		}
+
+		var customerId = "guess:" + properties.getCustomerIds().getOrDefault(alert.getSourceApp(),
+				"unknown@alert.iris-connect.de");
+		var text = alert.getText() + "<br /><br /><i>Client: " + alert.getClient() + "<br />App-Version: "
+				+ alert.getAppVersion() + "</i>";
```

Comment:
Suggestion: Put client, app-version and environment into the field 'note' in Ticket. This way the second call for tag could be omitted.

Reply:
Putting the environment in the tags was a suggestion from Andi, which Tim also liked. I don't know the system well enough yet, but tags can probably be selected specifically.

I had no success with `note` and looked again now where this entry ends up in the Zammad UI. However, I cannot find it.

Reply:
Therefore, I leave it as it is. 😉